pkg/apis/starrocks/v1/auto_scale.go

```diff
@@ -54,6 +56,28 @@ const (
 	AutoScalerV2 AutoScalerVersion = "v2"
 )
 
+// Complete completes the default value of AutoScalerVersion
+func (version AutoScalerVersion) Complete(major, minor string) AutoScalerVersion {
+	if version != "" {
+		return version
+	}
+	// operator choose a proper default hpa version by checking ths kubernetes version
+	// if kubernetes version > 1.25, use v2 version
```

Comment:
means >=1.26?

Reply:
yes

Reply:
let's make it `minorNumber >= 26` which is much intuitive.